NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage5/SettingsPage5ViewController.cs

```diff
@@ -50,6 +50,7 @@ public override void ViewDidLoad()
         private void InitAccessibilityStatementButton()
         {
             InitLinkButtonStyling(AccessibilityStatementButton, SettingsPage5ViewModel.SETTINGS_PAGE_5_ACCESSIBILITY_STATEMENT_BUTTON_TEXT);
+            AccessibilityStatementButton.AccessibilityTraits = UIAccessibilityTrait.Link;
```

Comment:
There were some issues with Accessibility VoiceOver, I basically set the whole paragraph with the link to be static text, so it won't be split. Does this work as intended?

Reply:
Yes. The only difference is that VoiceOver says "link" instead of "button"